hail/src/main/scala/is/hail/expr/ir/functions/Functions.scala

```diff
@@ -82,6 +83,39 @@ object IRFunctionRegistry {
       })
   }
 
+  def pyRegisterIRForServiceBackend(
+    ctx: ExecuteContext,
+    name: String,
+    typeParamStrs: Array[String],
+    argNames: Array[String],
+    argTypeStrs: Array[String],
+    returnType: String,
+    bodyStr: String
+  ): Unit = {
+    requireJavaIdentifier(name)
+
+    val typeParameters = typeParamStrs.map(IRParser.parseType).toFastIndexedSeq
+    val valueParameterTypes = argTypeStrs.map(IRParser.parseType).toFastIndexedSeq
+    val refMap = argNames.zip(valueParameterTypes).toMap
+    val body = IRParser.parse_value_ir(
```

Comment:
Why is this function `parse_value_ir` not camel case?

Reply:
Because they call it from Python, I presume.